code/game/atoms.dm

```diff
@@ -381,7 +381,7 @@ its easier to just keep the beam vertical.
 
 //Woo hoo. Overtime
 //All atoms
-/atom/proc/examine(mob/user, var/size = "")
+/atom/proc/examine(mob/user, var/size = "", var/show_name = TRUE)
```

Comment:
You have to and add this parameter to every `examine()` override now. Else you'll get runtime errors trying to use it.

Reply:
Would that not be done on a case-by-case basis depending on when it needs to be used?
`/obj/item/examine()` doesn't have the `var/size` parameter, for example.

Reply:
Since you can put any item on this trophy mount, you'll have to do it for every item.

I have no idea what's up with that `size` argument but it's probably the same boat.

Reply:
Oh, right, I forgot that there are items which override `examine()`.
Altering every override seems sub-optimal. Can you think of any alternative method of preventing the item's name from showing on `examine()`? Or would it be better to give up the description-mixing and just make it so that examining the trophy mount *only* examines the item on it?

Reply:
> Can you think of any alternative method of preventing the item's name from showing on examine()?

You can not use named arguments and just pass `""` into the second argument. That'd work.

Reply:
Alright, done.